crates/ruff_linter/src/rules/flake8_simplify/rules/needless_bool.rs

```diff
@@ -55,25 +55,31 @@ impl Violation for NeedlessBool {
 
     #[derive_message_formats]
     fn message(&self) -> String {
-        let NeedlessBool { condition, negate } = self;
-
-        if let Some(condition) = condition.as_ref().and_then(SourceCodeSnippet::full_display) {
+        if let Some(condition) = self
+            .condition
+            .as_ref()
+            .and_then(SourceCodeSnippet::full_display)
+        {
```

Comment:
Here I would probably keep the `let NeedlessBool { condition, negate } = self;` line, since it results in fewer lines of code overall, and it allows us to unpack multiple variables at once